packages/vue_frontend_rpc/src/components/Editor.vue

```diff
@@ -103,14 +85,19 @@ export default {
     },
     prepareTask(task) {
       this.task = task;
-      for (const question of task.content) {
+      const questions = [...task.content];
+      for (const question of questions) {
         for (const prop in question) {
           if (question[prop] === FUNCTION) {
-            const that = this;
-            question[prop] = this.getEvaluationFunction(that.rpc, question.name, prop);
+            var that = this;
+            question[prop] = async (...args) => {
+              const response = await that.rpc.invoke("evaluateMethod", [question.name, prop, args]);
```

Comment:
why didn't you use the `getEvaluationFunction` below? If you preferred to do it inline you can delete it from `methods` since it is not being used.